flashcards/web/Dockerfile

```diff
@@ -13,5 +13,6 @@ RUN npm run build
 
 FROM nginx:1.23.3 as production
```

Comment:
I deeply believe you don't want to expose this port as json server is dev only

Reply:
Fixed